langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibValueTest.java

```diff
@@ -172,6 +172,10 @@ public void testToString() {
         assertEquals(array.getRefValue(1).stringValue(), "4");
         assertEquals(array.getRefValue(2).stringValue(), "4");
         assertEquals(array.getRefValue(3).stringValue(), "4");
+        assertEquals(array.getRefValue(4).stringValue(), "error(\"Failed to get account balance\",details=true," +
+                "val1=NaN,val2=\"This Error\",val3={\"x\":\"AA\",\"y\":Infinity})");
+        assertEquals(array.getRefValue(5).stringValue(), "error FirstError (\"Reason1\",message=\"Test passing error " +
+                "union to a function\")");
```

Comment:
Shall we move these assertion to the bal file? Let's move the first 4 assertions in this method too.

Reply:
Fixed